ocs_ci/ocs/ui/page_objects/page_navigator.py

```diff
@@ -443,7 +443,7 @@ def nav_to_storageclients_page(self):
 
         logger.info("Navigate to Storage Client Page")
         self.choose_expanded_mode(mode=True, locator=self.page_nav["Storage"])
-        self.page_has_loaded(retries=120, sleep_time=10)
+        self.page_has_loaded(retries=300, sleep_time=30)
```

Comment:
this will make it even less possible to have the page loaded.
pls make sleep time 2-10 sec if you want that we pass page hash equality check within that function